src/parser/jobs/mnk/modules/evaluators/PlayersBuffedEvaluator.tsx

```diff
@@ -0,0 +1,87 @@
+import {Trans} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+import {Status} from 'data/STATUSES'
+import {EvaluatedAction, EvaluationOutput, WindowEvaluator} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {SEVERITY, SeverityTiers, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const DEFAULT_EXPECTED_PLAYER_COUNT = 8
+const DEFAULT_PLAYERS_MISSED_SEVERITY = {
+	1: SEVERITY.MINOR,
+}
+
+export interface PlayersBuffedEvaluatorOptions {
+	/**
+	 * This is used to determine how many players were buffed in a given window.
+	 * @param window The window to determine how many players were buffed
+	 * @returns The amount of players buffed in the window
+	 */
+	affectedPlayers: (window: HistoryEntry<EvaluatedAction[]>) => number
+	/**
+	 * Defaults to full party size.
+	 */
+	expectedPlayerCount?: number
```

Comment:
If this is optional, it doesn't have a default. Additionally, not all content uses 8, so I'd check if there's a way to grab party size sanely.